src/mocks/SensorData.ts

```diff
@@ -1,7 +1,7 @@
 export interface SensorData {
```

Comment:
Move types to `src/types`. Only mocked data should be in `src/mocks`, not types.